legend-engine-core/legend-engine-core-language-pure/legend-engine-language-pure-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/to/DEPRECATED_PureGrammarComposerCore.java

```diff
@@ -494,22 +516,33 @@ public String visit(PureInstanceClassMapping pureInstanceClassMapping)
         String pureFilter = "";
         if (pureInstanceClassMapping.filter != null)
         {
+            this.currentTabLevel += 1;
             pureInstanceClassMapping.filter.parameters = Collections.emptyList();
-            String filterString = pureInstanceClassMapping.filter.accept(this).replaceFirst("\\|", "");
-            pureFilter = getTabString(2) + "~filter " + filterString + "\n";
+            String filterString = pureInstanceClassMapping.filter.accept(Builder.newInstance().withCurrentTabLevel(this.currentTabLevel + 1).build()).replaceFirst("\\|", "");
+            pureFilter = getTabString(2) + "~filter \n " + getTabString(this.currentTabLevel + 1) + filterString + "\n";
```

Comment:
Is this intended? Does it make the code clearer to just move the filter down one level and indent?